config/libc.in

```diff
@@ -35,6 +35,10 @@ config LIBC_SUPPORT_THREADS_LT
 config LIBC_SUPPORT_THREADS_NONE
     bool
 
+config LIBC_SUPPORT_THREADS_POSIX
```

Comment:
Please keep the order the same as in the choice below - keeping "NONE" as the last.